src/main/java/org/elasticsearch/index/merge/scheduler/ConcurrentMergeSchedulerProvider.java

```diff
@@ -172,7 +172,9 @@ public void onRefreshSettings(Settings settings) {
                 logger.info("updating [{}] from [{}] to [{}]", MAX_MERGE_COUNT_KEY, ConcurrentMergeSchedulerProvider.this.maxMergeCount, maxMergeCount);
```

Comment:
wait we only pass the +1 to the call to the actual scheduler no? so these values there don't have the +1 at all